src/widgets/smallmap_widget.h

```diff
@@ -32,6 +32,8 @@ enum SmallMapWidgets : WidgetID {
 	WID_SM_ENABLE_ALL,     ///< Button to enable display of all legend entries.
 	WID_SM_DISABLE_ALL,    ///< Button to disable display of all legend entries.
 	WID_SM_SHOW_HEIGHT,    ///< Show heightmap toggle button.
+	WID_SM_SHOW_IND_NAMES, ///< Show industry names toggle button.
+	WID_SM_SHOW_IND_NAMES_SEL, ///< Container for the 'show industry names' button, which can be hidden.
```

Comment:
Adding a single space instead of shifting the alignment of other comments (in compliance of #12780)